### PR TITLE
Remove gallery controller and link Flickr group

### PR DESCRIPTION
**Pull request by dirkmueller**
The gallery controller was embedding the same Flickr gallery
via a Adobe Flash plugin, which is no longer a thing in 2022.

### CODE REVIEW

File: app/views/layouts/_footer.html.haml [hunk 1]
```diff
@@ -13,9 +13,9 @@
               %i.fas.fa-info
               About
           %li
-            = link_to(gallery_path) do
+            = link_to('https://www.flickr.com/groups/366038@N24/pool/', target: :_blank) do
```
Comment (hennevogel) [inline app/views/layouts/_footer.html.haml:16]:
```suggestion
            = link_to('https://www.flickr.com/groups/hackweek/') do
```

Comment (hennevogel) [reply app/views/layouts/_footer.html.haml:16]: Fiddling with peoples tabs is not really a thing in 2022 anymore ;-)

File: app/views/layouts/_footer.html.haml [hunk 2]
```diff
@@ -13,9 +13,9 @@
...
               %i.fas.fa-images
-              Pics
+              Picture Gallery
```
Comment (hennevogel) [inline app/views/layouts/_footer.html.haml:18]:
```suggestion
              Flickr Group
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: app/views/layouts/_footer.html.haml (hunk 1 above)
Comment (dirkmueller) [reply app/views/layouts/_footer.html.haml]: removed

**Review by hennevogel — APPROVED** [review]

**Pull request merged**